Core/Exercises/Solutions/SolutionRunnnerService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Text;
+using System.Text.Json;
+using Core.Exercises.Contracts.Services;
+using Core.Exercises.Models;
+using Core.Exercises.Services.TestRunners;
+using Microsoft.Extensions.Logging;
+
+namespace Core.Exercises.Solutions;
```

Comment:
Personally, believe that solution/test-runner would be more intuitive to have in its own folder, rather than laying under exercises.